modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/190_failure_store_redirection.yml

```diff
@@ -215,3 +220,100 @@ teardown:
       indices.delete:
         index: .fs-logs-foobar-*
   - is_true: acknowledged
+
+---
+"Ensure failure is redirected to correct failure store after a reroute processor":
+  - requires:
+      cluster_features: ["gte_v8.15.0"]
+      reason: "data stream failure stores REST structure changed in 8.15+"
```

Comment:
I think this is going to fail because it's actually broken in 8.15, but I'm not actually sure what the right approach would be here. Adding a capability to the REST API seems off, and we're not allowed to skip based on versions anymore. A node feature for this also seems off. Do people have any thoughts?